cpp/oneapi/dal/algo/basic_statistics/backend/cpu/compute_kernel.cpp

```diff
@@ -42,18 +41,17 @@ namespace daal_lom = daal::algorithms::low_order_moments;
 namespace interop = dal::backend::interop;
 
 template <typename Float, daal::CpuType Cpu>
-using daal_lom_batch_kernel_t =
+using dense_batch_kernel_t =
     daal_lom::internal::LowOrderMomentsBatchKernel<Float, daal_lom::defaultDense, Cpu>;
 
+template <typename Float, daal::CpuType Cpu>
+using sparse_batch_kernel_t =
+    daal_lom::internal::LowOrderMomentsBatchKernel<Float, daal_lom::fastCSR, Cpu>;
```

Comment:
The approach to dense and sparse DAAL kernels usage differs here and in SVM algorithm.
It would be better to align the implementations of basic statistics and SVM:

```cpp
template <typename Float, daal::CpuType Cpu, typename Method>
using batch_kernel_t =
    daal_lom::internal::LowOrderMomentsBatchKernel<Float, to_daal_method<Method>::value, Cpu>;
```